Sorry: IndentationError: expected an indented block after 'except' statement on line 58 (serial_temp_reader.py, line 61)
